workflow_sailor/Snakefile

```diff
@@ -23,14 +24,23 @@ def get_val_from_args(val_names):
     val = args[args.index(val_present) + 1]
     print('{} is {}'.format(val_present, val))
     
-    snakemake_directory = '/'.join(val.rstrip('/').split('/')[:-2])
-    print('STAMP directory is {}'.format(snakemake_directory))
+    if val_present == '--profile':  # read in profile doc to get snakedir
+        with open(os.path.join(val, 'config.yaml'), "r") as stream:
+            try:
+                return os.path.dirname(yaml.safe_load(stream)['snakefile'])
+            except yaml.YAMLError as exc:
+                print(exc)
 
-    return snakemake_directory
+    snakemake_directory = os.path.dirname(val)
+    print('STAMP directory is {}'.format(snakemake_directory))
+    if snakemake_directory.startswith('/'):  # abspath
+        return snakemake_directory
+    else:
+        return os.path.join(os.getcwd(), snakemake_directory)
 
 
 snakemake_directory = get_val_from_args(['snakefile', 's', 'profile'])
-scripts_directory='{}/workflow_sailor/scripts/'.format(snakemake_directory)
```

Comment:
I think changing this won't work... were you able to test this with all the use cases already?

Reply:
Yeah, we can go through it later today if you'll be in lab?